packages/webamp-modern-2/src/utils.ts

```diff
@@ -80,6 +80,20 @@ export function findLast<T>(
   }
 }
 
+
+export function getUrlQuery(location:Location, variable:string):string {
+  var query = location.search.substring(1);
+  var vars = query.split('&');
+  for (var i = 0; i < vars.length; i++) {
+      var pair = vars[i].split('=');
+      if (decodeURIComponent(pair[0]) == variable) {
+          return decodeURIComponent(pair[1]);
+      }
+  }
+  console.log('Query variable %s not found', variable);
+  return null;
```

Comment:
I think something like this would be simpler and more robust
```suggestion
  return new URL(location.href).searchParams.get(variable)
```